lib/rataextra-pipeline.ts

```diff
@@ -27,8 +29,27 @@ export class RataExtraPipelineStack extends Stack {
       authentication: oauth,
     });
 
-    const pipeline = new CodePipeline(this, 'Pipeline-RataExtra', {
-      pipelineName: 'pr-rataextra-' + config.stackId,
+    const artifactBucket = new Bucket(this, `s3-rataextra-pipeline-${config.stackId}`, {
+      autoDeleteObjects: true,
+      removalPolicy: RemovalPolicy.DESTROY,
+    });
+
+    const pipeline = new Pipeline(this, 'pipeline', {
+      artifactBucket: artifactBucket,
+      pipelineName: `pr-rataextra-${config.stackId}`,
```

Comment:
This should be named differently from the existing pipeline, because when it's merged to main, this name will conflict with the name of the existing pipeline. That's why the initial bit was named `cpl`.